README.md

```diff
@@ -540,16 +542,6 @@ const index = pc.index('test-index').namespace('ns1');
 await index.fetch(['1']);
 ```
 
-If needed, you can check the currently targeted index and namespace by inspecting the `target` property of an index object.
```

Comment:
Did this no longer work or is it kind of unnecessary?

Reply:
No longer works b/c property is private, sry for not adding context there!